compilers/concrete-compiler/compiler/include/concretelang/Dialect/FHELinalg/IR/FHELinalgOps.td

```diff
@@ -503,16 +505,16 @@ def FHELinalg_ApplyMultiLookupTableEintOp : FHELinalg_Op<"apply_multi_lookup_tab
     }];
 
     let arguments = (ins
-        Type<And<[TensorOf<[FHE_AnyEncryptedInteger]>.predicate, HasStaticShapePred]>>:$t,
+        Type<And<[TensorOf<[FHE_EncryptedIntegerType]>.predicate, HasStaticShapePred]>>:$t,
```

Comment:
That's a mistake due to my misunderstanding of the unconventional type name choice for unsigned integers. Usually (in C, C++, Rust), unsignedness is explicitly marked, not signedness.

I'm rolling back this change and renaming `FHE_EncryptedIntegerType` to `FHE_EncryptedUnsignedIntegerType`. I'm not changing the `eint` mnemonic though, since it might have many external uses. However, I'd consider renaming `eint` to `euint`, to avoid that third parties run into a similar confusion.

Reply:
I'm not in favor of renaming at least in this PR, because this should change the API with the frontend. But I agree it could be good to rename it later and to avoid the confusion the mnemonic too